src/test/java/guitests/HistoryCommandTest.java

```diff
@@ -0,0 +1,24 @@
+package guitests;
+
+import org.junit.Test;
+
+/**
+ * Gui tests for history command
+ */
+public class HistoryCommandTest extends ToLuistGuiTest {
+    @Test
+    public void viewHistory() {
+        String command = "history";
+        commandBox.runCommand(command);
+        assertResultMessage("\n" + command + "\n2 commands displayed.");
+    }
+
+    @Test
+    public void addTaskAndViewHistory() {
+        String command1 = "add task";
+        commandBox.runCommand(command1);
+        String command2 = "history";
+        commandBox.runCommand(command2);
+        assertResultMessage("\n" + command1 + "\n" + command2 + "\n3 commands displayed.");
+    }
```

Comment:
Need more tests, like running data-mutating and non-mutating commands together (and maybe some undo and redo) and then see the history